test/unit/number/format.js

```diff
@@ -151,6 +151,10 @@ QUnit.test( "should allow rounding", function( assert ) {
 	assert.equal( format( pi, properties( "0.20", en ) ), "3.20" );
 	assert.equal( format( pi, properties( "0.5", en ) ), "3.0" );
 	assert.equal( format( pi, properties( "0.1", en ) ), "3.1" );
+
+	// Handle inaccurate floating point arithmetics like 0.00015 * 10000 = 1.49999999999999.
+	// See #376.
+	assert.equal( format( 0.00015, properties( "0.0001", en ) ), "0.0002" );
```

Comment:
This change fixes inaccurate floating point arithmetics related to rounding decimals only. its usage is not a special option, it's always used when formatting a number/currency with decimal digits. Any rounding that is performed by number/currency formatting is performed by the same rounding method that I'm testing above. Did I answer you? Let me ask + answer some other related questions myself...

*Can other parts of the globalize code eventually run into the same rounding problem reported here and we are not testing it?*

No. Any rounding is performed by the method I'm testing above. This is true at least for any number code. On date parsing, the native Math.round is used directly to parse milliseconds and the native Math.floor is used directly to parse centuries, which I am not sure the same precaution should be taken. Any test in mind for those?

*Can this test be placed somewhere else?*

Yes. I've placed this test under "Number Format ld allow rounding". But, this test could be placed either under maximum fraction digits, or under minimum fraction, or under significant digits formatting, etc. Formatting a decimal in any of those forms would eventually hit the same problem. But, testing it in one place is enough to catch it.

*Other globalize code indirectly uses the rounding code that has been changed. Is everything else working just fine like before?*

Yes, it should. At least, it didn't regress in any test. All tests are still passing, including `1234 * 0.0001 = 0.12340000000000001` case tested by previous existing "Number Format should format significant digits".

Please, just let me know if you still have any question.